workflows/plugins/include/utils/alerts.py

```diff
@@ -14,10 +15,20 @@ def task_failure_alert(context):
         logger.error("Cannot send email.")
         return
 
+    base_url = os.environ.get("AIRFLOW_BASE_URL")
+    task_instance = context["task_instance"]
+    parsed_log_url = urlparse(task_instance.log_url)
```

Comment:
so by default airflow returns the log_url as with `localhost` as a host?

Reply:
yes by default it returns it with localhost:8080. On the other hand I just saw that, https://airflow.apache.org/docs/apache-airflow/stable/configurations-ref.html#base-url . Let me try to replace it with the appropriate varialble, maybe it does the job out of the box for us

Reply:
that will do the trick . Will close this pr and re-open another one